### PR TITLE
[Tests-Only] Adjust TrashbinContext asserts and exceptions for then and given steps

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This PR includes the adjustments in the asserts and exceptions for the then and given steps respectively in the TrashbinContext file.

## Related Issue
#36810 

## How Has This Been Tested?
CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/TrashbinContext.php [hunk 1]
```diff
@@ -81,7 +81,9 @@ public function userHasEmptiedTrashbin($user) {
 		$response = $this->emptyTrashbin($user);
 
 		Assert::assertEquals(
-			204, $response->getStatusCode()
+			204,
+			$response->getStatusCode(),
+			__METHOD__ . "Expected status code was '204' but got '" . $response->getStatusCode() . "'"
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TrashbinContext.php]:
```suggestion
			__METHOD__ . " Expected status code was '204' but got '" . $response->getStatusCode() . "'"
```

File: tests/acceptance/features/bootstrap/TrashbinContext.php [hunk 2]
```diff
@@ -450,7 +455,11 @@ public function asFileOrFolderExistsInTrash($user, $path) {
 			}
 		}
 
-		Assert::assertTrue($found);
+		Assert::assertTrue(
+			$found,
+			__METHOD__
+			. "Could not find expected resource '$path' in the trash"
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TrashbinContext.php]:
```suggestion
			. " Could not find expected resource '$path' in the trash"
```

File: tests/acceptance/features/bootstrap/TrashbinContext.php (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/TrashbinContext.php]: fixed

File: tests/acceptance/features/bootstrap/TrashbinContext.php (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/features/bootstrap/TrashbinContext.php]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**